app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -370,8 +370,6 @@ ShortFormApplicationController = (
   $scope.checkAfterLiveInTheNeighborhood = (preference) ->
     # preference is either neighborhoodResidence or antiDisplacement
     if ShortFormApplicationService.applicationHasPreference(preference)
-      # Neighborhood provides automatic liveInSf preference
-      ShortFormApplicationService.copyNeighborhoodToLiveInSf(preference)
```

Comment:
@kaptron Ah I see yes because we need to sync up the member too! Good catch, will implement.